homeassistant/components/switch/flux.py

```diff
@@ -111,7 +112,8 @@ def update(call=None):
         """Update lights."""
         flux.flux_update()
 
-    hass.services.register(DOMAIN, name + '_update', update)
+    service_name = slugify("{} {}".format(name, 'update'))
+    hass.services.register(DOMAIN, service_name, update)
```

Comment:
Yeah, I always thought it was strange, but I was just trying to fix this for now.